Sources/Defaults/Defaults.swift

```diff
@@ -65,6 +65,18 @@ public final class Defaults {
 			suite.removeObject(forKey: key)
 		}
 	}
+	
+	public func clear<T: Codable>(key: Defaults.Key<T>, suite: UserDefaults = .standard) {
```

Comment:
Can you add a documentation comment for this and the other clear method?

Reply:
I think it would be useful to accept multiple keys.

Reply:
Sure, documentation is useful here.

Reply:
> I think it would be useful to accept multiple keys.

I agree, but I would rather implement it in an additional method.
Something like this: `public func clear<T: Codable>(keys: [Defaults.Key<T>]`

What's your opinion?

Reply:
Do it like https://github.com/sindresorhus/Defaults/pull/21#issuecomment-519684656